src/scripts/create_dutch_cola.py

```diff
@@ -0,0 +1,38 @@
+"""Create the Dutch CoLA test set and upload it to the HF Hub."""
+
+from datasets import Dataset, DatasetDict, load_dataset
+from huggingface_hub import HfApi
+from requests import HTTPError
+
+
+def main() -> None:
+    """Create the DutchSocial-mini sentiment dataset and upload it to the HF Hub."""
+    # Define the base download URL
+    repo_id = "GroNLP/dutch-cola"
+
+    # Download the dataset
+    dataset = load_dataset(path=repo_id, token=True)
+    del dataset["train"]
```

Comment:
As mentioned in my overall comment, we need train/val/test splits for all datasets. I'd recommend 1024/256/2048 here, ideally sampled from the original train/val/test splits.